react/src/App.tsx

```diff
@@ -1,9 +1,17 @@
+import { IconButton } from '@material-ui/core';
+import { Close } from '@material-ui/icons';
+import { SnackbarKey, SnackbarProvider } from 'notistack';
 import React, { useState, useEffect } from 'react';
 import { createMuiTheme, ThemeProvider } from '@material-ui/core';
 
 import LoginForm from './pages/Login';
 import Navigation from './pages/Navigation';
 
+const notistackRef = React.createRef<any>();
+const onClickDismiss = (key: SnackbarKey) => () => {
+    notistackRef.current.closeSnackbar(key);
```

Comment:
I believe the typing to `createRef` is `SnackbarProvider` and then you can do `current!.closeSnackbar` to assert not null since this is only called from the IconButton click?